evals/seed/remove.py

```
@@ -0,0 +1,209 @@
+        rows = page.results if hasattr(page, "results") else page
+        for customer in rows or []:
+            if (customer.name or "").strip().casefold() in (CUSTOMER_NAME.casefold(), "acme"):
+                # Only delete if we seeded or created during this run (tracked or name match + run).
+                tracked = {
+                    obj.get("id") for obj in (ctx.get("workspace_objects") or []) if obj.get("kind") == "customer"
+                }
+                if str(customer.id) in tracked or ctx.get("customer") is None:
+                    # Avoid deleting long-lived Acme if we pre-seeded and tracked it — still delete tracked.
+                    if str(customer.id) in tracked or not ctx.get("customer"):
+                        ctx.setdefault("workspace_objects", []).append({"kind": "customer", "id": customer.id})
+    except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)